sqlmesh/core/snapshot/definition.py

```diff
@@ -771,10 +766,7 @@ def version_get_or_generate(self) -> str:
     def physical_schema(self) -> str:
         if self.physical_schema_ is not None:
             return self.physical_schema_
-        schema = exp.to_table(self.name).db
-        if not schema:
-            schema = c.DEFAULT_SCHEMA
```

Comment:
I don't see this logic anywhere now. We do need to support a default schema since the model name might just contain a table name without a schema

Reply:
I take this back. Found this logic in the model definition.

Reply:
https://github.com/TobikoData/sqlmesh/blob/c091cd2301a0c707c86d30d78e8dd0d4ba63d088/sqlmesh/core/model/definition.py#L555